charts/modules/infra/aws-eks/README.md

```diff
@@ -15,28 +15,28 @@
-| `global.argocdIAMRole`          | Argo CD IAM role to grant cross account assume role permissions                                             | `argo-cd`                        |
-| `global.crossplaneNamespace`    | Crossplane namespace for storing the EKS cluster config                                                     | `infra-crossplane`               |
-| `global.crossplaneIAMRole`      | Crossplane IAM Role                                                                                         | `crossplane-provider-aws`        |
-| `global.providerConfigRefName`  | AWS Provider config ref name of the local crossplane instance                                               | `crossplane-provider-aws`        |
-| `global.eksVersion`             | Default EKS Version to use                                                                                  | `1.25`                           |
-| `global.eksNodeVersion`         | EKS node version                                                                                            | `1.25.11-20230816`               |
-| `global.teamAdminIAMRole`       | Grant EKS admin permissions to this IAM Role                                                                | `infra-admins`                   |
+| Name                                | Description                                                                                                 | Value                            |
+| ----------------------------------- | ----------------------------------------------------------------------------------------------------------- | -------------------------------- |
+| `global.chartNameOverride`          | Overrides the chart name.                                                                                   | `""`                             |
+| `global.releaseNameOverride`        | Overrides the release name.                                                                                 | `""`                             |
+| `global.tags`                       | Define common tags for all IAC and app resources generated by this chart.                                   | `{}`                             |
+| `global.labels`                     | Define common labels for all IAC and app resources generated by this chart.                                 | `{}`                             |
+| `global.annotations`                | Define common annotations for all IAC and app resources generated by this chart.                            | `{}`                             |
+| `global.awsAccountId`               | Default aws account id for crossplane aws provider resources. Quotes are important, value must be a string. | `0123456789`                     |
+| `global.awsMgmtAccountId`           | Management AWS account that runs the main mgmt cluster + argocd + crossplane.                               | `0123456789`                     |
```

Comment:
`awsBackupAccountId` is missing.

Please run `helm-chart-generate-doc`

Reply:
Generated the new README to include awsBackupAccountId